README.md

```diff
@@ -3,14 +3,10 @@ banana-rdf
 
 [![Build Status](https://secure.travis-ci.org/w3c/banana-rdf.png)](http://travis-ci.org/w3c/banana-rdf) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/w3c/banana-rdf?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-The current published version 0.8.4 for scala 2.12 is to be found on the [http://bblfish.net/work/repo/releases](http://bblfish.net/work/repo/releases) repository.
+The current published version 0.8.5-SNAPSHOT for scala 2.13 is to be found on the Sonatype repository under groupId [net/bblfish/rdf/](https://oss.sonatype.org/content/repositories/snapshots/net/bblfish/rdf/).
 
 ```scala
-val banana = (name: String) => "org.w3" %% name % "0.8.4" excludeAll (ExclusionRule(organization = "org.scala-stm"))
-
-//add the bblfish-snapshots repository to the resolvers
-
-resolvers += "bblfish-snapshots" at "http://bblfish.net/work/repo/releases"
```

Comment:
Instead the Sonatype resolver is required here.

Reply:
ah yes. The - and the Red is what was removed :-)

Reply:
Ok, I was was not sure about this :+1:

Reply:
It looks like it has not been deployed yet, which is why you would need the resolver still. I will look into that.

Reply:
Ah I see SNAPSHOT releases are not published further so one does need to add

```scala
resolvers += ("snapshots" at
    "https://oss.sonatype.org/content/repositories/snapshots")
```

in order to use them. As I had not published to sonatype I was not aware of the reason for needing those before.

Is that the extra line you need in your build file? I'll add that to the README if so.

Reply:
Yes, or an equivalent setting. I used exactly this:

`resolvers += Resolver.sonatypeRepo("snapshots")`